libs/image/website/src/lib/image.tsx

```diff
@@ -26,9 +26,12 @@ export const imageToSquareImageItems = (image: FullImageFragment): ImageItem[] =
 
 export const ImageWrapper = styled('img')<{aspectRatio: number}>`
   max-width: 100%;
-  width: 100%;
   height: auto;
   aspect-ratio: auto ${({aspectRatio}) => aspectRatio};
+  // Sets a max height for images so that they will not be too tall
+  // and makes them not distorted
+  max-height: 80lvh;
```

Comment:
first time I see lv 😮 looks like it might solve some issues, especially on mobile